modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/object/definitions/portlet/action/test/dependencies/test-object-definition-state-manager.json

```diff
@@ -0,0 +1,419 @@
+			"label": {
+				"en_US": "External Reference Code"
+			},
+			"localized": false,
+			"name": "externalReferenceCode",
+			"objectFieldSettings": [
+			],
+			"readOnly": "false",
+			"readOnlyConditionExpression": "",
+			"required": false,
+			"state": false,
+			"system": true,
+			"type": "String",
+			"unique": false
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"indexed": true,
+			"indexedAsKeyword": true,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "ID"
+			},
+			"localized": false,
+			"name": "id",
+			"objectFieldSettings": [
+			],
+			"readOnly": "true",
+			"readOnlyConditionExpression": "",
+			"required": false,
+			"state": false,
+			"system": true,
+			"type": "Long",
+			"unique": false
+		},
+		{
+			"DBType": "Date",
+			"businessType": "Date",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Modified Date"
+			},
+			"localized": false,
+			"name": "modifiedDate",
+			"objectFieldSettings": [
+			],
+			"readOnly": "true",
+			"readOnlyConditionExpression": "",
+			"required": false,
+			"state": false,
+			"system": true,
+			"type": "Date",
+			"unique": false
+		},
+		{
+			"DBType": "String",
+			"businessType": "Text",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Status"
+			},
+			"localized": false,
+			"name": "status",
+			"objectFieldSettings": [
+			],
+			"readOnly": "true",
+			"readOnlyConditionExpression": "",
+			"required": false,
+			"state": false,
+			"system": true,
+			"type": "String",
+			"unique": false
+		},
+		{
+			"DBType": "String",
+			"businessType": "Picklist",
+			"defaultValue": "open",
+			"externalReferenceCode": "88bf6ec4-ec8d-e209-3412-7e1b0190b328",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "TestObjectImport"
+			},
+			"listTypeDefinitionExternalReferenceCode": "LIST_APPLICATION_STATES",
+			"localized": false,
+			"name": "testobjectimport",
+			"objectFieldSettings": [
+				{
+					"name": "stateFlow",
+					"value": {
+						"objectStates": [
+							{
+								"key": "open",
+								"objectStateTransitions": [
+									{
+										"key": "underReview"
+									}
+								]
+							},
+							{
+								"key": "underReview",
+								"objectStateTransitions": [
+									{
+										"key": "open"
+									},
+									{
+										"key": "approved"
+									}
+								]
+							},
+							{
+								"key": "approved",
+								"objectStateTransitions": [
+								]
+							}
+						]
+					}
+				},
+				{
+					"name": "defaultValueType",
+					"value": "inputAsValue"
+				},
+				{
+					"name": "defaultValue",
+					"value": "open"
+				}
+			],
+			"readOnly": "false",
+			"readOnlyConditionExpression": "",
+			"required": true,
+			"state": true,
+			"system": false,
+			"type": "String",
+			"unique": false
+		}
```

Comment:
Hey @victorKammerer can we just put this part of the **json** inside of the `test-object-definition.json` used [here](https://github.com/liferay-objects/liferay-portal/blob/b3fe5897be00934775f11dc84627d39c9b18a6a1/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java#L49)?